### DISCUSSION TITLE
web analytics

### DISCUSSION THREAD

**Discussion by kaciccone** [knowledgefutures/pubpub#2852] [General]
Is there a way to get web traffic data for a PubPub community?

**Comment by jeffpooley** [comment]
Hi Karen,

You can view analytics at the Community, Collection, and Pub level. There is an Impact tab in each of those Dashboard views. (see screenshot).

Cheers! Jeff

<img width="1366" alt="screenshot" src="https://github.com/pubpub/pubpub/assets/6372329/abbc1698-547d-4c14-a130-eda7f7408235">

**Comment by kaciccone** [comment]
Thank you!

On Tue, Sep 26, 2023 at 4:58 PM Jeff Pooley ***@***.***>
wrote:

> Hi Karen,
>
> You can view analytics at the Community, Collection, and Pub level. There
> is an Impact tab in each of those Dashboard views. (see screenshot).
>
> Cheers! Jeff
> [image: screenshot]
> <https://user-images.githubusercontent.com/6372329/270795071-abbc1698-547d-4c14-a130-eda7f7408235.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pubpub/pubpub/discussions/2852#discussioncomment-7116961>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE32ILQO7HJXXHMMAWFBWITX4M6WZANCNFSM6AAAAAA5IIZZWU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Karen Ciccone
Lead Librarian for Public Science
NC State University Libraries | http://www.lib.ncsu.edu
***@***.***
http://orcid.org/0000-0002-8861-9857

All electronic mail messages in connection with State business which are
sent to or received by this account are subject to the NC Public Records
Law and may be disclosed to third parties.
